domains/*.ricardo-inc.is-local.org.json

```diff
@@ -0,0 +1,20 @@
+{
+    "description": "Bussnis website",
+
+    "domain": "is-local.org",
+    "subdomain": "ricardo-inc",
+
+    "owner": {
+        "repo": "https://hosting-everyone.github.io/my-website/",
+        "email": "rifoneud@gmail.com"
+    },
+
+    "record": {
+        "A": ["194.24.161.232", "185.199.108.153", "185.199.109.153", "185.199.110.153", "185.199.111.153"]
+        "AAAA": ["2606:50c0:8000::153", "2606:50c0:8001::153", "2606:50c0:8002::153", "2606:50c0:8003::153"]
+        "NS": ["cory.ns.cloudflare.com", "mia.ns.cloudflare.com"]
```

Comment:
Cannot have NS records with other records. Doesn't work with wildcard records. No reason is provided.